internal/cache/text.go

```diff
@@ -7,37 +7,37 @@ import (
 )
 
 var (
-	sizeCache = map[sizeEntry]fontMetric{}
-	sizeLock  = sync.RWMutex{}
+	fontSizeCache = map[fontSizeEntry]fontMetric{}
+	fontSizeLock  = sync.RWMutex{}
 )
 
 type fontMetric struct {
-	bound    fyne.Size
+	size     fyne.Size
 	baseLine float32
 }
 
-type sizeEntry struct {
+type fontSizeEntry struct {
 	text  string
 	size  float32
 	style fyne.TextStyle
 }
 
 // GetFontSize looks up a calculated size required for the specified text parameters.
 func GetFontSize(text string, size float32, style fyne.TextStyle) (fyne.Size, float32) {
-	ent := sizeEntry{text, size, style}
-	sizeLock.RLock()
-	ret, ok := sizeCache[ent]
-	sizeLock.RUnlock()
+	ent := fontSizeEntry{text, size, style}
+	fontSizeLock.RLock()
+	ret, ok := fontSizeCache[ent]
+	fontSizeLock.RUnlock()
 	if !ok {
 		return fyne.Size{Width: 0, Height: 0}, 0
 	}
-	return ret.bound, ret.baseLine
+	return ret.size, ret.baseLine
```

Comment:
I hadn't seen carefully this function before. Should be better to rename this to GetFontMetrics? and add named return parameters for clarity?

Reply:
Sure, done